extension/src/wallet/index.ts

```diff
@@ -11,7 +11,7 @@ export class Server {
   async setWallet(wallet: string) {
     this.wallet = wallet;
     await linera;
-    this.client = await new linera.Client(await linera.Wallet.fromJson(wallet));
+    this.client = await new linera.Client({} as linera.Wallet); // Replace with actual wallet initialization
```

Comment:
`fromJson` is removed from the `linera-web` client on `linera-protocol`.

Reply:
I think this is officially broken now; can we get a TODO?